### PR TITLE
Rules for JS Audit

### PR DESCRIPTION
**Pull request by ankit-privado**

### CODE REVIEW

File: rules/sinks/third_parties/sdk/sequelize/javascript.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.SDK.Sequelize
```
Comment (HiteshMah-Jan) [inline rules/sinks/third_parties/sdk/sequelize/javascript.yaml:7]: Sequelize seems to be an ORM. can we move this to database section. Also the pattern added will not get much matches, need some impl similar to Django ORM for this. Please remove this rule.

Comment (HiteshMah-Jan) [inline rules/sinks/third_parties/sdk/sequelize/javascript.yaml:7]: Sequelize seems to be an ORM. can we move this to database section. Also the pattern added will not get much matches, need some impl similar to Django ORM for this. Please remove this rule

Comment (ankit-privado) [reply rules/sinks/third_parties/sdk/sequelize/javascript.yaml:7]: yes for now i will remove this.

Comment (ankit-privado) [reply rules/sinks/third_parties/sdk/sequelize/javascript.yaml]: done.

**Review by HiteshMah-Jan — APPROVED** [review]

**Review by HiteshMah-Jan — APPROVED** [review]

**Pull request merged**